peekaboo/ruleset/rules.py

```diff
@@ -526,8 +541,9 @@ def evaluate(self, sample):
 
             if result:
                 return self.result(result,
-                                   _("A rule classified the sample as %s")
-                                   % result,
+                                   _("The rule (%s) classified the sample as %s")
+                                   % (self.expressions[i].replace('\n', ' ').replace('\r', ''),
```

Comment:
Aren't we leaking internal configuration here and potentially confusing the users by presenting them with our weird rule syntax?

Reply:
Yes, we do both. We leak slightly more information here than we do with the other rules.
I think we should have at least the number of the rule inside the virus mail. We need to balanace acceptance / usability / information leak

Reply:
I removed this change and keep it for future